docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+services:
+  mtproto-proxy:
+    #image: telegrammessenger/proxy:latest
+    build:
+      context: .
+      args:
+        MTPROTO_REPO_URL: ${MTPROXY_REPO_URL}
+    container_name: mtproto-proxy
+    volumes:
+      - ./volumes/config:/data
+    restart: unless-stopped
+    ports:
+      - 8443:8889/udp
+      - 8443:8889/tcp
+    environment:
+      - TAG
+      - SECRET
+      - SECRET_COUNT
```

Comment:
What is this ENV?

Reply:
Sorry, I think that is an old copy past. I implemented this have a lot of time but never did the PR. Looking the code sounds that is unused environments, except the `SECRET`. I  removed all unused.